backend/rebutify/settings.py

```diff
@@ -65,6 +68,16 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
+# Domains that can access the API
+CORS_ALLOWED_ORIGINS = [
+    "https://rebutify.org",
+    "http://localhost:3000",
+    "http://0.0.0.0:3000",
+]
+
+# Normalizing URLs for front-end (https://docs.djangoproject.com/en/4.0/ref/middleware/#django.middleware.common.CommonMiddleware)
+APPEND_SLASH = False
```

Comment:
Good change, could you move this to another branch?

From the linked article
> Both of these options are meant to normalize URLs. The philosophy is that each URL should exist in one, and only one, place. Technically a URL foo.com/bar is distinct from foo.com/bar/ – a search-engine indexer would treat them as separate URLs – so it’s best practice to normalize URLs.

Since SEO is important, could you change this to True?
Maybe add PREPEND_WWW too